prove/Develop05/ChecklistGoal.cs

```diff
@@ -24,7 +24,7 @@ public override bool CheckCompletionStatus()
 
     public override void DisplayProgressMessage()
     {
-        if (CheckCompletionStatus())
+        if (_numOfTimesAccomplished == _duration)
```

Comment:
I would normally use an inequality here.  Checking for equality can bite you if for some reason the count jumps.

Reply:
Will be worked on later.
Thanks